vsc_pdf_template_transformer/lib/models/tpl_text.dart

```diff
@@ -65,26 +64,28 @@ class TplText implements wb.WidgetBuilder {
       maxLines: maxLines,
       overflow: overflow,
     );
-
-    print('--- value style: ${value.text.style!.color!.toHex()} ------');
     return value;
   }
 
-  String? _evaluateInput(String? text, Map<String, dynamic> data) {
-    if (text != null) {
-      print(text);
+  String _evaluateInput(TplString text, Map<String, dynamic> data) {
```

Comment:
Do you see how this code is basically duplicated between TplSizedBox and TplText? See my comment about moving this to TplString.